packages/api/src/brevo-api/brevo-api.utils.ts

```diff
@@ -1,5 +1,26 @@
+import { ErrorModel } from "@getbrevo/brevo";
 import * as http from "http";
 
-export function isErrorFromBrevo(error: unknown): error is { response: http.IncomingMessage } {
-    return typeof error === "object" && error !== null && "response" in error && error.response instanceof http.IncomingMessage;
+export function isErrorFromBrevo(error: unknown): error is { response: http.IncomingMessage; body: ErrorModel } {
+    return (
+        typeof error === "object" &&
+        error !== null &&
+        "response" in error &&
+        error.response instanceof http.IncomingMessage &&
+        "body" in error &&
+        typeof error.body === "object" &&
+        error.body !== null &&
+        "code" in error.body &&
+        typeof error.body.code === "string" &&
+        "message" in error.body &&
+        typeof error.body.message === "string"
+    );
+}
+
+export function handleBrevoError(error: unknown): never {
+    if (isErrorFromBrevo(error)) {
+        throw new Error(error.body.message);
+    } else {
+        throw new Error();
```

Comment:
Shouldn't this re-throw the inital error? 
Eg.: https://github.com/vivid-planet/comet-brevo-module/pull/104/files#diff-f7b10559b572b094baf502b0beda3119a997258f6396a41344f75de5bddbe41dR148

Reply:
806b2daba08c0b5f5fb1bcd57d9da40c16d2fca5